index.js

```diff
@@ -45,6 +47,7 @@ const pLimit = concurrency => {
 		},
 		clearQueue: {
 			value: () => {
+				queue.forEach(({reject}) => reject(new Error('queue cleared before function was invoked')));
```

Comment:
Use a `for-of` loop.

and:

```suggestion
				queue.forEach(({reject}) => reject(new Error('Queue cleared before function was invoked')));
```